ludwig/serve.py

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+
+from ludwig.contrib import contrib_command
+from ludwig.utils.print_utils import logging_level_registry
+import json
+import os
+import tempfile
+from starlette.datastructures import UploadFile
+
+from ludwig.api import LudwigModel
+
+
+logger = logging.getLogger(__name__)
+
+FILE_TYPES = set([
+    'image',
+    'audio',
+    'video'
+])
+
+
+def start_server(
+    model_path,
+    logging_level,
+    host,
+    port
+):
+
+    from fastapi import FastAPI
```

Comment:
Since this is the main functionality anyway, why not do imports at the top of the file?